apps/emqx_management/src/emqx_management.app.src

```diff
@@ -1,6 +1,6 @@
 {application, emqx_management,
  [{description, "EMQ X Management API and CLI"},
-  {vsn, "4.3.3"}, % strict semver, bump manually!
+  {vsn, "4.3.4"}, % strict semver, bump manually!
```

Comment:
emqx_management 4.3.3 It hasn't been released yet, so you don't need to modify the version number

Reply:
it's already bumped, but not released yet.